cli/flags.rs

```diff
@@ -1362,6 +1367,29 @@ fn permission_args_parse(flags: &mut Flags, matches: &clap::ArgMatches) {
   }
 }
 
+fn watch_arg<'a, 'b>() -> Arg<'a, 'b> {
+  Arg::with_name("watch")
+    .long("watch")
+    .min_values(0)
+    .takes_value(true)
+    .use_delimiter(true)
+    .require_equals(true)
+    .help("Restart the server on file change")
+}
+
+fn watch_arg_parse(flags: &mut Flags, matches: &clap::ArgMatches) {
+  if let Some(watch_p) = matches.values_of("watch") {
+    let raw_watch_paths: Vec<PathBuf> = watch_p.map(PathBuf::from).collect();
+
+    if raw_watch_paths.is_empty() {
+      flags.watch_paths = vec![PathBuf::from(".")];
```

Comment:
In `watch_arg` it's defined that `--watch` can take no parameters - I guess in that case we'd want to watch current working directory for all files matching `.js/.jsx/.ts/.tsx` extension. If so then `watch_arg_parse` needs to be changed to check `matches.is_present()` - otherwise `--watch` flag without paths will be ignored.

Reply:
Verified that it works as expected